tpl/strings/init.go

```diff
@@ -112,6 +112,34 @@ func init() {
 			},
 		)
 
+		ns.AddMethodMapping(ctx.TrimLeft,
+			nil,
+			[][2]string{
+				{`{{ strings.TrimLeft "++Batman++" "+" }}`, `Batman++`},
```

Comment:
It would be better (in my eyes) if all examples (where possible) were on "pipe form", i.e. 

```
{`{{ "++Batman++" | strings.TrimLeft  "+" }}`, `Batman++`},
```